CONTRIBUTING.adoc

```diff
@@ -0,0 +1,56 @@
+= Contributing a blog post to Open Liberty
```

Comment:
Maybe the title should be more specific and say something about signing the PRs. This title is very similar to the title of the README so I was initially confused about the purpose of this file

Reply:
I was wondering about that. It's a standard file name that I think we should stick with for consistency. I think actually, we should move most of the instructions from the README to the CONTRIBUTING file. I'll have another look tomorrow.